modules/virt-importing-vm-prerequisites.adoc

```diff
@@ -43,7 +43,7 @@ Importing a VMware VM into {VirtProductName} has the following prerequisites:
 
 * You must have access to an image registry.
 * You must create a VMware Virtual Disk Development Kit (VDDK) image, push it to an image registry, and add it to the `v2v-vmware` ConfigMap.
-* The VMware virtual machine must be powered off during import.
+* The VMware virtual machine must be powered off before you import it.
```

Comment:
At first, I wasn't sure whether "it" was the VM or the VDDK image from the previous prereq.

One thought is to change the structure to something like, "You must power off the VMware VM before you import it" to stay consistent with the previous prereqs. (And if so, change the following one as well, "You must have sufficient storage space....") This might help to clarify the subject.

Reply:
Agree. Changes made

Reply:
I also fixed a typo in the storage matrix file.